[AzCopyV10] Include edge-cases for exit codes (#96398)
Azure/azure-content: articles/storage/common/storage-use-azcopy-troubleshoot.md

If the exit code is `2-panic`, then check the log file exists. If the file doesn't exist, file a bug or reach out to support.

> If the exit code is any other non-zero exit code, it may be a exit code from the system (e.g. OOM kill). Check your operating system documentation for special exit codes.

Reviewer: ```suggestion
If the exit code is any other non-zero exit code, it may be an exit code from the system. For example, OOMKilled.  Check your operating system documentation for special exit codes.
```
'eg.' isn't supposed to be used in Docs.

Author: For the sake of learning, what's the exact reasoning behind this?

Reviewer: @adreed-msft - I'm not sure of the reasoning, but my guess is because it's less clear and doesn't localize very well.  Here's the MS Style Guide entry: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=33667
